test/CrowdsaleGenTest.js

```diff
@@ -346,17 +399,21 @@ contract('LifCrowdsale Property-based test', function(accounts) {
   });
 
   let checkCrowdsaleState = async function(state, crowdsaleData, crowdsale) {
-    assert.equal(state.crowdsalePaused, await crowdsale.paused.call());
-    assert.equal(_.sumBy(state.purchases, (b) => b.tokens), help.lifWei2Lif(parseFloat(await crowdsale.tokensSold.call())));
+    assert.equal(state.crowdsalePaused, await crowdsale.paused());
+    assert.approximately(
+      _.sumBy(state.purchases, (b) => b.tokens),
+      parseFloat(help.lifWei2Lif(parseFloat(await crowdsale.tokensSold()))),
+      0.00001
```

Comment:
Where does this difference come from? Also it would be nice to use a higher precision (more decimals, let's say 0.00000000001), or is this the lowest precision before the test breaks?